tools/clang/include/clang/AST/HlslTypes.h

```diff
@@ -382,9 +382,8 @@ bool IsHLSLLineStreamType(clang::QualType type);
 bool IsHLSLTriangleStreamType(clang::QualType type);
 bool IsHLSLStreamOutputType(clang::QualType type);
 bool IsHLSLResourceType(clang::QualType type);
-bool IsHLSLNumeric(clang::QualType type);
```

Comment:
I moved this to the source file because it's only used by IsHLSLNumericUserDefinedType, and I think it's badly named. I wouldn't expect it to match a struct with numeric-typed members.